db/src/main/java/io/lantern/db/DB.kt

```diff
@@ -535,14 +535,12 @@ class DB private constructor(
     /**
      * Returns a SharedPreferences backed by this db.
      *
-     * @param schema - preference keys are stored in this named schema
      * @param fallback - an optional fallback SharedPreferences to use for values that aren't found in the db
      */
     fun asSharedPreferences(
-        schema: String,
         fallback: SharedPreferences? = null
     ): SharedPreferencesAdapter {
-        return SharedPreferencesAdapter(this.withSchema(schema), fallback)
+        return SharedPreferencesAdapter(this, fallback)
```

Comment:
Just curious about why the removal of the schema string reference?

Reply:
This allows us to use the existing database. `withSchema` creates a derived DB that has its own listeners and doesn't notify the ones on the original database.